cmd/load-generator/main.py

```diff
@@ -158,25 +117,10 @@ def main():
     namespace = sys.argv[2]
     pr_number = sys.argv[3]
 
-    host = os.environ.get('KUBERNETES_SERVICE_HOST')
-    port = os.environ.get('KUBERNETES_PORT_443_TCP_PORT')
-    url = 'https://%s:%s' % (host, port)
-
-    kube_token = open(kube_token_path, 'r').read()
-
-    req_kwargs = {
-        'headers': {'Content-type': 'application/json', 'Authorization': 'Bearer ' + kube_token},
-        'verify': ca_cert_path,
-    }
-
     config = yaml.load(open("/etc/loadgen/config.yaml", 'r').read())
 
     print("loaded configuration")
 
-    if sys.argv[1] == "scaler":
-        Scaler(config["scaler"], url, req_kwargs).run()
-        return
-
     if sys.argv[1] == "querier":
```

Comment:
You can remove this check. We need not check for presence of `"scalar"` in sys.argv.
The "querier" argument can be removed from the config file since only that is there now. https://github.com/prometheus/prombench/blob/02b50cc659cca89db4a1392ddd510c5fdf1bd674/components/prombench/manifests/benchmark/6_loadgen.yaml#L114